src/utils/strings.js

```diff
@@ -10,4 +10,41 @@ function snakeCaseToStartCase(str) {
     .join(' ');
 }
 
-export { snakeCaseToStartCase };
+/**
+ * Formats multiple session numbers into a string
+ * @param {Array<string>} sessionNumbers - Array of session numbers
+ * @returns {string} Formatted string of session numbers
+ */
+function formatMultipleSessionNumbers(sessionNumbers) {
+  if (sessionNumbers.length === 2) {
+    return `${sessionNumbers[0]} and ${sessionNumbers[1]}`;
+  }
+
+  const sortedNumbers = sessionNumbers.map(Number).sort((a, b) => a - b);
+  let result = `${sortedNumbers[0]}`;
+  let rangeLength = 1;
+
+  for (let i = 1; i < sortedNumbers.length; i++) {
+    const current = sortedNumbers[i];
+    const prev = sortedNumbers[i - 1];
+
+    if (current === prev + 1) {
+      rangeLength++;
+    } else {
+      if (rangeLength > 2) {
+        result += `...${prev}`;
+      }
+      result += `, ${current}`;
```

Comment:
if every third number is not part of range does this algorithm print this?
`1...2,4...5,7...8` That seems odd.

Reply:
actually nope. reread it. it just removes the second number in range.
```
formatMultipleSessionNumbers([1,2,4,5,7,8])
// '1,4,7'
```